backend/src/nodes/nodes/image_adjustment/dither.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+import cv2
+import numpy as np
+from sanic.log import logger
+
+from . import category as ImageAdjustmentCategory
+from ...impl.dithering import bayer_filter
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import ImageInput, NumberInput, DropDownInput, get_number_type
+from ...properties.outputs import ImageOutput
+
+
+@NodeFactory.register("chainner:image:dither")
+class DitherNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Apply a Bayer ordered dithering algorithm."
+        self.inputs = [
+            ImageInput(),
+            NumberInput("Colors per channel", minimum=2, default=4),
+            DropDownInput(
+                input_type=get_number_type(0, 16, 0),
```

Comment:
That function wasn't meant to be exported by the module. Sorry about that.

And this input type isn't meant to be a number type. It's meant to be a distinct type that is incompatible with any other type. But this distinct type has to be declared in `chainner-scope.ts`. Frankly, I wouldn't recommend doing it that way. I would recommend using an `EnumInput` + `option_labels` ([example](https://github.com/chaiNNer-org/chaiNNer/blob/4f4caaca5f3f49769e53620617eb92fff91d0e90/backend/src/nodes/nodes/image_dimension/resize_to_side.py#L106)) because `EnumInput` automatically handles typing for you.